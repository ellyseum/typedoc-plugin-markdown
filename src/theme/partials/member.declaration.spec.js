describe(`member.declaration.hbs`, () => {
  test('should compile', () => {
    const context = {
      name: 'numbers',
      comment: {},
      sources: [],
    };

    const result = global.compileTemplate('member.declaration.hbs', context);
    expect(result).toMatchSnapshot();
  });

  test('should compile with type declarations', () => {
    const context = {
      name: 'specular',
      sources: [],
      type: {
        type: 'reflection',
        declaration: {
          sources: [],
          signatures: [],
        },
      },
    };

    const result = global.compileTemplate('member.declaration.hbs', context);
    expect(result).toMatchSnapshot();
  });
});
