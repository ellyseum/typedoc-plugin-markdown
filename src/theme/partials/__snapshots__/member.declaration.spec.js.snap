// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`member.declaration.hbs should compile 1`] = `
"[member.declaration.title]
[member.sources]
[comment]"
`;

exports[`member.declaration.hbs should compile with type declarations 1`] = `
"[member.declaration.title]
[member.sources]
[comment]

#### Type declaration

[parameter]"
`;
